### ISSUE TITLE
Warnings with config file

### ISSUE THREAD

**Issue by Halleck45** [Vectorface/dunit#8]
Hi,

I copy paste the content of the default configfile in `.dunitconfig`. The command `./vendor/bin/dunit` outputs:

> WARNING: Config file could not be read. Falling back to built-in defaults

Full output:

```
PHP Deprecated:  Comments starting with '#' are deprecated in /home/data/www/jeff/misc/halstead/.dunitconfig on line 1 in phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/DunitCommand.php on line 267
PHP Stack trace:
PHP   1. {main}() /home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit:0
PHP   2. require() /home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit:10
PHP   3. Symfony\Component\Console\Application->run() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/dunit.php:7
PHP   4. Symfony\Component\Console\Application->doRun() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/vendor/symfony/console/Symfony/Component/Console/Application.php:126
PHP   5. Symfony\Component\Console\Application->doRunCommand() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/vendor/symfony/console/Symfony/Component/Console/Application.php:195
PHP   6. Symfony\Component\Console\Command\Command->run() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/vendor/symfony/console/Symfony/Component/Console/Application.php:874
PHP   7. Vectorface\Dunit\DunitCommand->execute() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/vendor/symfony/console/Symfony/Component/Console/Command/Command.php:252
PHP   8. Vectorface\Dunit\DunitCommand->getCurrentOptions() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/DunitCommand.php:136
PHP   9. Vectorface\Dunit\DunitCommand->mergeConfigFileIntoOptions() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/DunitCommand.php:180
PHP  10. parse_ini_file() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/DunitCommand.php:267

Deprecated: Comments starting with '#' are deprecated in /home/data/www/jeff/misc/halstead/.dunitconfig on line 1 in phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/DunitCommand.php on line 267

Call Stack:
    0.0001     228880   1. {main}() /home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit:0
    0.0030     355480   2. require('phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/dunit.php') /home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit:10
    0.0121    1777552   3. Symfony\Component\Console\Application->run() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/dunit.php:7
    0.0143    2113536   4. Symfony\Component\Console\Application->doRun() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/vendor/symfony/console/Symfony/Component/Console/Application.php:126
    0.0143    2114256   5. Symfony\Component\Console\Application->doRunCommand() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/vendor/symfony/console/Symfony/Component/Console/Application.php:195
    0.0143    2114800   6. Symfony\Component\Console\Command\Command->run() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/vendor/symfony/console/Symfony/Component/Console/Application.php:874
    0.0145    2117440   7. Vectorface\Dunit\DunitCommand->execute() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/vendor/symfony/console/Symfony/Component/Console/Command/Command.php:252
    0.0145    2117704   8. Vectorface\Dunit\DunitCommand->getCurrentOptions() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/DunitCommand.php:136
    0.0146    2120392   9. Vectorface\Dunit\DunitCommand->mergeConfigFileIntoOptions() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/DunitCommand.php:180
    0.0146    2120632  10. parse_ini_file() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/DunitCommand.php:267

PHP Warning:  syntax error, unexpected BOOL_TRUE in /home/data/www/jeff/misc/halstead/.dunitconfig on line 7
 in phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/DunitCommand.php on line 267
PHP Stack trace:
PHP   1. {main}() /home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit:0
PHP   2. require() /home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit:10
PHP   3. Symfony\Component\Console\Application->run() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/dunit.php:7
PHP   4. Symfony\Component\Console\Application->doRun() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/vendor/symfony/console/Symfony/Component/Console/Application.php:126
PHP   5. Symfony\Component\Console\Application->doRunCommand() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/vendor/symfony/console/Symfony/Component/Console/Application.php:195
PHP   6. Symfony\Component\Console\Command\Command->run() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/vendor/symfony/console/Symfony/Component/Console/Application.php:874
PHP   7. Vectorface\Dunit\DunitCommand->execute() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/vendor/symfony/console/Symfony/Component/Console/Command/Command.php:252
PHP   8. Vectorface\Dunit\DunitCommand->getCurrentOptions() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/DunitCommand.php:136
PHP   9. Vectorface\Dunit\DunitCommand->mergeConfigFileIntoOptions() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/DunitCommand.php:180
PHP  10. parse_ini_file() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/DunitCommand.php:267

Warning: syntax error, unexpected BOOL_TRUE in /home/data/www/jeff/misc/halstead/.dunitconfig on line 7
 in phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/DunitCommand.php on line 267

Call Stack:
    0.0001     228880   1. {main}() /home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit:0
    0.0030     355480   2. require('phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/dunit.php') /home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit:10
    0.0121    1777552   3. Symfony\Component\Console\Application->run() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/dunit.php:7
    0.0143    2113536   4. Symfony\Component\Console\Application->doRun() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/vendor/symfony/console/Symfony/Component/Console/Application.php:126
    0.0143    2114256   5. Symfony\Component\Console\Application->doRunCommand() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/vendor/symfony/console/Symfony/Component/Console/Application.php:195
    0.0143    2114800   6. Symfony\Component\Console\Command\Command->run() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/vendor/symfony/console/Symfony/Component/Console/Application.php:874
    0.0145    2117440   7. Vectorface\Dunit\DunitCommand->execute() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/vendor/symfony/console/Symfony/Component/Console/Command/Command.php:252
    0.0145    2117704   8. Vectorface\Dunit\DunitCommand->getCurrentOptions() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/DunitCommand.php:136
    0.0146    2120392   9. Vectorface\Dunit\DunitCommand->mergeConfigFileIntoOptions() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/DunitCommand.php:180
    0.0146    2120632  10. parse_ini_file() phar:///home/data/www/jeff/misc/halstead/vendor/vectorface/dunit/bin/dunit/src/DunitCommand.php:267

WARNING: Config file could not be read. Falling back to built-in defaults.
```

My environment:
```
PHP 5.4.6-1ubuntu1.8 (cli) (built: Apr  4 2014 01:28:36) 
Copyright (c) 1997-2012 The PHP Group
Zend Engine v2.4.0, Copyright (c) 1998-2012 Zend Technologies
    with Xdebug v2.2.1, Copyright (c) 2002-2012, by Derick Rethans
```

Since PHP 5.3, `#` and simplde quotes are deprecated : [documentation of parse_ini_file](http://php.net/manual/fr/function.parse-ini-file.php)

**Comment by danbruce** [comment]
Thank you for reporting this. The issue has been fixed and v2.0.1 has been released as a result.

**Comment by Halleck45** [comment]
Ok, good job
